src/dbos-runtime/config.ts

```diff
@@ -127,11 +127,18 @@ function prettyPrintAjvErrors(validate: ValidateFunction<unknown>) {
   }).join(', ');
 }
 
+export interface ParseOptions {
+  port?: number;
+  loglevel?: string;
+  configfile?: string;
```

Comment:
I just realized configFile is deprecated, but is still used in parseConfigFile. Should we remove it here?

Reply:
For being in a hurry, I think fine either way.